Neos.Flow/Tests/Unit/Reflection/MethodReflectionTest.php

```diff
@@ -44,4 +45,28 @@ public function getParametersReturnsFlowsParameterReflection($dummyArg1 = null,
             self::assertEquals(__CLASS__, $parameter->getDeclaringClass()->getName());
         }
     }
+
+    public function classAndMethodWithAnnotations()
+    {
+        return [
+            [ClassWithAnnotatedMethod::class, 'methodWithTag', ['skipcsrfprotection' => []]],
+            [ClassWithAnnotatedMethod::class, 'methodWithTagAndComment', ['skipcsrfprotection' => ['Some comment']]],
+            [ClassWithAnnotatedMethod::class, 'methodWithAnnotation', ['flow\skipcsrfprotection' => []]],
+            [ClassWithAnnotatedMethod::class, 'methodWithAnnotationAndComment', ['flow\skipcsrfprotection' => ['Some comment']]],
+            [ClassWithAnnotatedMethod::class, 'methodWithAnnotationArgument', ['flow\validate' => ['"foo"']]],
+            [ClassWithAnnotatedMethod::class, 'methodWithAnnotationArgumentAndComment', ['flow\validate' => ['"foo"']]],
+            [ClassWithAnnotatedMethod::class, 'methodWithMultipleAnnotationArguments', ['flow\ignorevalidation' => ['argumentName="foo", evaluate=true']]],
+            [ClassWithAnnotatedMethod::class, 'methodWithMultipleAnnotationArgumentsAndComment', ['flow\ignorevalidation' => ['argumentName="foo", evaluate=true']]],
+        ];
+    }
+
+    /**
+     * @test
+     * @dataProvider classAndMethodWithAnnotations
+     */
+    public function commentsAfterAnnotationShouldBeIgnored($class, $method, $expected)
+    {
+        $method = new Reflection\MethodReflection($class, $method);
+        self::assertEquals($method->getTagValues());
```

Comment:
```suggestion
        self::assertEquals($method->getTagsValues());
```
Coding by GH is doomed to fail for such things...